src/grid/column.component.ts

```diff
@@ -196,6 +200,8 @@ export class IgxColumnComponent implements AfterContentInit {
     protected _hidden = false;
     protected _index: number;
 
+    private _defaultMinWidth = "88";
```

Comment:
If we decide to go with the hardcoded minWidth probably it will be better to give it some more space. Currently if there are icons in the header and we resize the column to minimal allowed size the column caption disappears.